src/libraries/System.IO.Hashing/src/System.IO.Hashing.csproj

```diff
@@ -30,6 +29,7 @@ System.IO.Hashing.XxHash32</PackageDescription>
     <Reference Include="System.Runtime" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
+    <Compile Include="System\IO\Hashing\BitOperations.cs" />
     <PackageReference Include="System.Buffers" Version="$(SystemBuffersVersion)" />
```

Comment:
Since VS doesn't merge References and Compiles in the same ItemGroup node I usually don't, either.  But, I also dislike duplicated logic.  So... le shrug :smile:.